kubernetes/utils/delete_from_yaml.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import re
+from os import path
+
+import yaml
+
+from kubernetes import client
+
+
+def delete_from_yaml(k8s_client, yaml_file, verbose=False,
+                     namespace="default", **kwargs):
+    """Input:
```

Comment:
better to describe what the function does before the Input:

Reply:
okay will modify and push